test/runtests.jl

```diff
@@ -83,4 +83,8 @@ using Infinity, Infinity.Utils, Test
     @inferred sign(InfExtended{Int32}(∞))
   end
 
-end
\ No newline at end of file
+  @testset "conversions" begin
+    @test convert(Infinite, InfExtended{Int}(∞)) === ∞
```

Comment:
we may as well test we didn't forget the `signbit`?
```suggestion
    @test convert(Infinite, InfExtended{Int}(∞)) === ∞
    @test convert(Infinite, InfExtended{Int}(-∞)) === -∞
```